### ISSUE TITLE
How to stack bars over each other

### ISSUE THREAD

**Issue by michaelw85** [coroo/nova-chartjs#42]
By default a stacked bar chart adds all values. According to the [documentation](https://www.chartjs.org/docs/latest/charts/bar.html#stacked-bar-chart) to overlap the values a configuration like this is needed:

```js
var stackedBar = new Chart(ctx, {
    type: 'bar',
    data: data,
    options: {
        scales: {
            xAxes: [{
                stacked: true
            }],
            yAxes: [{
                stacked: true
            }]
        }
    }
});
```

How would I enter this config into nova-chartjs?

**Comment by coroo** [comment]
Hi @michaelw85 , 

Actually we set those for default value for stacked chart, so that we can use it more easily by choose between [stacked-chart](https://coroo.github.io/nova-chartjs/#/?id=stacked-chart) or [bar-chart](https://coroo.github.io/nova-chartjs/#/?id=bar-chart) and no need to do enter that config manually again.

Thank you.

**Comment by michaelw85** [comment]
@coroo Thanks for your quick reply!
I just realised I completely misinterpreted the ChartJS docs and I was looking for the wrong option in your documentation.

Let me try to explain again what I'm trying to accomplish.
For example if you have 2 datasets with value **10** and one with value **5** and this would be displayed as a stacked bar I would now see a total bar height of **15**. What I would like to see is a total of 10 with 5 on top of/in front of the 10 bar (which I thought the "stacked" option would do)

**Comment by coroo** [comment]
Hi @michaelw85 , sorry if my explanation before didn't answer you question, can you see my screenshot video here:
https://www.loom.com/share/ec5cadf35e0b47989113a49eae9af499

Please let me know again if this one is not what you meaning for. Thankyou

**Comment by michaelw85** [comment]
Maybe a picture explains it better:
![image](https://user-images.githubusercontent.com/3629094/81800027-7bf7aa80-9512-11ea-904c-272e5fdcd833.png)

**Comment by coroo** [comment]
Oh, lol. I got your point now. Sorry and thank you @michaelw85 .

But from documentation and sample from chartjs e.g. https://www.chartjs.org/samples/latest/charts/bar/stacked.html or https://www.chartjs.org/samples/latest/charts/bar/stacked-group.html only capture `current` condition. 

Also, as your first explanation, we already add stacked code:
```
            xAxes: [{
                stacked: true
            }],
            yAxes: [{
                stacked: true
            }]
```
at [this file](https://github.com/coroo/nova-chartjs/blob/master/resources/js/components/StackedChart.vue) but the result will be as `current condition`.

**Comment by michaelw85** [comment]
Thank you for helping me out even though this seems to be more of a chartjs question.
I think it's impossible what I would like to accomplish, going to go over the examples again to see if one has the setup I'm looking for.

**Comment by michaelw85** [comment]
Ok best I could do was combine graphs ending up with:
![image](https://user-images.githubusercontent.com/3629094/81811188-84f17780-9524-11ea-89ba-06395afc4c3f.png)

```json
(new BarChart())
            ->title('Organization time logged vs budget')
            ->animations([
                'enabled' => true,
                'easing' => 'easeinout',
            ])
            ->series([
                [
                    'type' => 'line',
                    'pointStyle' => 'line',
                    'steppedLine' => 'middle',
                    'fill' => true,
                    'label' => 'Budget',
                    'borderColor' => "rgba(3,169,244, 0.2)",
                    'borderWidth' => 1,
                    'clip' => 0,
                    'backgroundColor' => "rgba(3,169,244, 0.2)",
                    'data' => array_values($budgets),
                ],
                [
                    'type' => 'bar',
                    'label' => 'Spent',
                    'backgroundColor' => 'rgba(3,169,244, 1)',
                    'data' => array_values($logs),
                ]
            ])
            ->options([
                'xaxis' => [
                    'categories' => array_keys($budgets),
                ],
                'legend' => [
                    'display' => true,
                    'position' => 'right',
                ],
                'btnRefresh' => true
            ])
            ->width('2/3')
            ->onlyOnDetail();
```

I've noticed that PHP is enforcing fill false for a linechart and RGBA is not working for area charts because it uses a method to brighten the border color which only accepts hex values.

**Comment by coroo** [comment]
Hi @michaelw85 , thank you for sharing your workaround. 
Also yes, we uses a method to brighten the border color. 

So, is there anything again that we can help you with this issue?

**Comment by michaelw85** [comment]
@coroo For now this setup is good enough for me.

If you want I could make 2 new separate tickets to describe the issues I encountered (and could probably offer a PR if you want after discussing if or how it should change)

**Comment by coroo** [comment]
Hi @michaelw85 ,

Yes please create separate issue. All feedback related this package is always welcome.

Again thank you :)

**Comment by michaelw85** [comment]
I just quickly created 2 issues, I'm a bit short on time so the information might not be enough please let me know if addition info is needed. I will close this ticket :)

**Comment by xoco70** [comment]
Hi @michaelw85, I just had the error with Chart.JS, and could do what you want to achieve easily with:

```
```yAxes: [{
                stacked: false
            }]

Nevertheless, I doesn't seem to change anything in nova
